### PR TITLE
Contract Types Plugin E2E Tests - rewrite

### PR DESCRIPTION
**Pull request by michaelkernaghan**
# 🌮 Taqueria PR

## 🪁 Description
Close# 1659
Rewrite e2e tests for contract types plugin
Setup for local loading of plugins by relative path traversal

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [ ] ⛱️ I have completed this PR template in full and updated the title appropriately
- [ ] ⛵ My code builds cleanly, and I have manually tested the changes
- [ ] 🏄‍♂️ Another team member has built this branch and done manual testing on the change
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🦀 Automated tests have been written and added to this PR
- [ ] 🐬 I have commented my code, particularly in hard-to-understand areas
- [ ] 🤿 Corresponding changes have been made to all documentation
- [ ] 🐚 Required changes to the VScE have been made
- [ ] 🪸 Required updates to scaffolds have been made
- [ ] 🚢 The release checklist has been completed

## 🛩️ Summary of Changes
rewrite e2e tests for contract types plugin

## 🎢 Test Plan

_Please describe the testing strategy and plan for this PR. Keep this lightweight and anticipate any testing challenges_

## 🛸 Type of Change

- [ ] 🧽 Chore ➾
- [ ] 🛠️ Fix ➾
- [ ] ✨ Feature ➾
- [ ] 👷 Refactor ➾
- [ ] 🧪 Pre-Release ➾
- [ ] 🚀 Release ➾

### CODE REVIEW

File: tests/e2e/taqueria-plugin-contract-types.spec.ts [hunk 1]
```diff
@@ -1,333 +1,289 @@
...
-
-			const generateTypesHelpContentsGenTypes = await exec(`taq gentypes --help --projectDir=${taqueriaProjectPath}`);
-			expect(generateTypesHelpContentsGenTypes.stdout).toBe(contents.helpContentsContractTypesPluginSpecific);
-		} catch (error) {
-			throw new Error(`error: ${error}`);
-		}
-	});
+	test.skip('1635 - generate types offers contextual help', async () => {
+		const { execute, spawn, cleanup } = await prepareEnvironment();
+		const { waitForText } = await spawn('taq', 'init test-project');
```
Comment (jchenche) [inline tests/e2e/taqueria-plugin-contract-types.spec.ts:24]: Btw do you mean to install the "core" plugin from NPM or local? If local, then provide `--debug` flag. If not this is fine.

This applies to all `taq init`s in e2e tests.

Comment (jchenche) [reply tests/e2e/taqueria-plugin-contract-types.spec.ts:24]: Note: "core" plugin is installed automatically upon `taq init`. That's why I wanted to confirm.

Comment (michaelkernaghan) [reply tests/e2e/taqueria-plugin-contract-types.spec.ts:24]: The plugin loading will now be local only by relative path traversal

**Review by egarson — APPROVED** [review]
LGTM

**Review by jchenche — APPROVED** [review]

**Pull request merged**